py3_test_reqs.txt

```diff
@@ -0,0 +1,2 @@
+six==1.10.0
+git+https://github.com/python/mypy.git@5d2dfce8cf6b9a0a7ce8fe8e3cb5866094459f8c
```

Comment:
Thinking about this a bit more, the `tools/` top-level directory is intended to be scripts used in the Zulip development environment but not useful in production (that's what distinguishes it from `scripts/`, which are scripts needed in production).  We already have `scripts/setup` for code related to setting up a production environment, so I think it'd be reasonable to make a `tools/setup` directory and move provision.py and its friends into there.

I think it's fine to put this new file in there, but we can handle migrating the rest to a `tools/setup` directory as a separate PR -- no reason to block getting this done on that process.